tests/test_interface.rb

```diff
@@ -682,9 +712,8 @@ def test_negotiate_auto_ethernet
     # Some platforms/interfaces/versions do not support negotiation changes
     begin
       interface.negotiate_auto = false
-    rescue => e
-      skip('Skip test: Interface type does not allow config change') if
-        e.message[/requested config change not allowed/]
+    rescue RuntimeError => e
+      speed_change_disallowed(e.message)
```

Comment:
Hmmm. You're right. I tested on n30,n31,n9k. Maybe you have a different linecard. Which testbed? i'll fix it.